test/rules/object-literal-sort-keys/shorthand-first/test.ts.lint

```diff
@@ -0,0 +1,79 @@
+const a = 'a'
+const c = 'c'
+const x = {}
+
+const fail1 = {
+  a,
+  b: 'b',
+  c,
+  ~  [shorthandErr % ('c')]
```

Comment:
what happens in the following example? I expect no errors because a `...x` spread should reset the sorting.

```ts
const fail1 = {
  a,
  b: 'b',
  ...x,
  c,
  d: 'd',
};
```

I think there needs to be a test case like this; I don't see one currently in this file.

Reply:
@adidahiya Yes this would give no error. The [pass5](https://github.com/brunolemos/tslint/blob/7b1223ecac22a43235d3cbd42f79b52a8ba8ac1a/test/rules/object-literal-sort-keys/shorthand-first/test.ts.lint#L69-L76) seems to be the closest to that for now:

```js
const pass5 = {
  d: 'd',
  ...x,
  c,
  ...x,
  a,
  b: 'b',
}
```

Reply:
ah, I missed pass5; that looks sufficient.